.unreleased/template.release_notes.j2

```diff
@@ -0,0 +1,16 @@
+{#
+ # Variables:
+ #   release_current = the release version that has being builded
+ #   release_previous = previous release version
+ #   release_date = date of the release in format YYYY-MM-DD
+ #}
+## {{ release_current }} ({{ release_date }})
+
+This release contains performance improvements and bug fixes since 
+the {{ release_previous }} release. We recommend that you upgrade at the next
+available opportunity.
+
+In addition, it includes these noteworthy features:
+* feature 1
+* feature 2
+* feature N
```

Comment:
We should probably add a flag in the release files which ones are major and should end up as highlighted.